src/main/java/ua/ek/steps/bdd/tablets/filters/TabletsFilterTestBdd.java

```diff
@@ -0,0 +1,97 @@
+package ua.ek.steps.bdd.tablets.filters;
+
+import cucumber.api.java.After;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.WebDriver;
+import ua.ek.steps.HomeStep;
+import ua.ek.steps.tablets.TabletsListStep;
+import ua.ek.steps.tablets.TabletsStep;
+import ua.ek.steps.tablets.filters.PriceFilterStep;
+import ua.ek.utils.*;
+
+public class TabletsFilterTestBdd {
+
+    private WebDriver driver;
+    private Helper helper;
+
+    private HomeStep homeStep;
+    private TabletsStep tabletsStep;
+    private PriceFilterStep priceFilterStep;
+    private TabletsListStep tabletsListStep;
+
+    public TabletsFilterTestBdd() {
+        InitRemoteDrivers initRemoteDrivers = new InitRemoteDrivers();
+        initRemoteDrivers.initWithDefaultValues();
+        driver = initRemoteDrivers.getDriver();
+        helper = new Helper(driver);
```

Comment:
this structure used in every BDD class. maybe you can take it into a separate entity?

Reply:
fixed